install.sh

```diff
@@ -89,7 +89,7 @@ echo " - This script will install the following packages:"
 echo "   - pytorch=$PYTORCH_VERSION"
 echo "   - pytorch-cuda=$CUDA_VERSION"
 echo "   - torchvision"
-if [ $INSTALL_TORCH_GEOMETRIC = "true" ]; then
+if [[ $INSTALL_TORCH_GEOMETRIC == "true" ]]; then
```

Comment:
Why this change? the first version is less bash-specific so seems better

Reply:
There's a `[: =: unary operator expected` error currently. We can do `[ "$INSTALL_TORCH_GEOMETRIC" = "true" ]` if you prefer non-bash. Fwiw, all modern shells support double bracket.